lib/File/Find/Rule.pm

```diff
@@ -563,11 +576,11 @@ sub in {
         my $discarded;
         return unless ' . $fragment . ';
         return if $discarded;
-        if ($relative) {
-            push @found, $relpath if $relpath ne "";
+        if ($relative and ($relpath ne "") ) {
```

Comment:
I don't see how the change is going to ignore `$relative`. The current logic requires two conditions to be _true_ in order to push the `$relpath`. I've consolidated both those conditions into a single if-condition. Given that the `normalize` behavior added yet another condition of _which_ `$relpath` will be pushed (default vs. canonpath-ed), there seemed to be too many conditions in the same statement.

If we leave the current structure, and add the new `normalize` condition, it might end up looking like:

```
if ($relative){
    push @found, $normalize ? File::Spec->canonpath($relpath) : $relpath if $relpath ne "";
}
```

Which, to me, is somewhat unreadable. I am open to changing this to the full long form with multiple if blocks.

Also, the current test for `relative` passes successfully. If there is a condition where this won't work, we need a test for it.